draft-ietf-quic-recovery.md

```diff
@@ -1407,15 +1409,23 @@ Invoked when an ACK frame with an ECN section is received from the peer.
 Invoked from DetectLostPackets when packets are deemed lost.
 
 ~~~
-   InPersistentCongestion(largest_lost_packet):
+   InPersistentCongestion(lost_packets):
```

Comment:
I'd add an early return at the very beginning to check if lost_packets is false to avoid questions about whether first() and last() are properly defined.

Reply:
I think this is already checked before calling `OnPacketsLost()`?
```
  if (!lost_packets.empty()):
    OnPacketsLost(lost_packets)
```

Reply:
Agreed, thanks.